diss.tex

```diff
@@ -328,20 +328,20 @@
 \institution{The University of British Columbia}
 \campus{Vancouver}
 
-\faculty{The Faculty of XXX}
+\faculty{The Faculty of Graduate and Postdoctoral Studies}
 \department{Basket Weaving}
 \submissionmonth{April}
 \submissionyear{2192}
 
 % details of your examining committee
-\examiningcommittee{John Smith, Materials Engineering}{Supervisor}
-\examiningcommittee{Mary Maker, Materials Engineering}%
+\examiningcommittee{John Smith, Professor, Materials Engineering, \textsc{UBC}}{Supervisor}
+\examiningcommittee{Mary Maker, Professor, Materials Engineering, \textsc{UBC}}%
     {Supervisory Committee Member}
-\examiningcommittee{Nebulous Name, Department}{Supervisory Committee Member}
-\examiningcommittee{Magnus Monolith, Other Department}{Additional Examiner}
+\examiningcommittee{Nebulous Name, Position, Department}{Supervisory Committee Member}
```

Comment:
They didn’t want an institution here?

Reply:
Ah, that's my bad for missing that! Added the institution placeholders.